CMakeLists.txt

```diff
@@ -84,7 +84,7 @@ set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} ${FLAGS_ALL}")
 # https://community.arm.com/developer/tools-software/tools/b/tools-software-ides-blog/posts/compiler-flags-across-architectures-march-mtune-and-mcpu
 # We check specifically for arm64 (e.g., Apple's M1) or aarch64 (e.g., Raspberry Pi 4), as these are the two ARM architectures we have tested so far.
 # Match for "arm" on your own risk to include other ARM architures.
-if (CMAKE_SYSTEM_PROCESSOR MATCHES "arm64|aarch64")
+if (CMAKE_SYSTEM_PROCESSOR MATCHES "arm64|aarch64|ppc64le")
```

Comment:
Could you add a comment/extend the one on ARM builds? Is it similar to ARM and arch native works for GCC but not for clang?